src/CalculatorUnitTests/CopyPasteManagerTest.cpp

```diff
@@ -825,9 +825,12 @@ namespace CalculatorUnitTests
                                      L"12^2",
                                      L"-12.12^-2",
                                      L"61%99"
-                                     L"-6.1%99" };
+                                     L"-6.1%99",
+                                     L"1.1111111111111111111111111111111e+1142" };
         String
             ^ negativeInput[] = { L"abcdef", L"xyz",   L"ABab", L"e+234", L"123456789123456781234567890123456" /*boundary condition: greater than 32 digits*/,
+                                  L"11.1111111111111111111111111111111e+1142",
+                                  L"1.1e+10001", /*boundary condition:exponent greater than 5 digits*/
```

Comment:
Can you add another unit test for the 0.xxxe+yyy case (eg. 0.11111111111111111111111111111111111+111111)? This case is handled separately in CopyPasteManager.cpp, as shown in the diff.

Reply:
Will add this boundary case. Here both exponent and non exponent components exceed limits.